src/board_builder/structures/charuco_board.py

```diff
@@ -0,0 +1,30 @@
+class CharucoBoard:
```

Comment:
If this is the same as the class in `detector.structures`, just move it to the `common.structures` location